packages/babel-plugin-transform-vite-meta-glob/src/index.ts

```diff
@@ -85,7 +87,9 @@ export default function viteMetaGlobBabelPlugin({
           t.isBooleanLiteral(eagerOption[0].value)
         ) {
           const cwd = nodePath.dirname(sourceFile)
-          const globPaths = globSync(args[0].value, { cwd, dotRelative: true }).sort()
+          const globPaths = globSync(args[0].value, { cwd, dotRelative: true })
+            .sort()
+            .map((globPath) => globPath.replace(/\\/g, '/'))
```

Comment:
**thoughts (non-blocking):** at what what point should we extract a function for this to ensure this and the above stay in sync?